korangar/src/world/entity/mod.rs

```diff
@@ -251,12 +252,26 @@ fn get_entity_part_files(script_loader: &ScriptLoader, entity_type: EntityType,
         )
     }
 
-    fn player_head_path(sex_sprite_path: &str, head_id: usize) -> String {
-        format!("ÀÎ°£Á·\\¸Ó¸®Åë\\{}\\{}_{}", sex_sprite_path, head_id, sex_sprite_path)
+    fn player_head_path(sex_sprite_path: &str, hair_id: usize) -> String {
+        format!("ÀÎ°£Á·\\¸Ó¸®Åë\\{}\\{}_{}", sex_sprite_path, hair_id, sex_sprite_path)
     }
 
+    let male_hair_ids = vec![2, 2, 1, 7, 5, 4, 3, 6, 8, 9, 10, 12, 11];
+    let female_hair_ids = vec![2, 2, 4, 7, 1, 5, 3, 6, 12, 10, 9, 11, 8];
+
+    let hair_style_id = match (sex, hair_id) {
+        (Sex::Male, 0..12) => male_hair_ids[hair_id],
+        (Sex::Male, hair_id) => hair_id,
+        (Sex::Female, 0..12) => female_hair_ids[hair_id],
```

Comment:
Looks like the 12 here could be replaced by `MALE_HAIR_LOOKUP.len()` and `FEMALE_HAIR_LOOKUP.len()` respectively.